src/BigBlueButton.php

```diff
@@ -451,7 +451,7 @@ public function getRecordingTextTracksUrl($getRecordingTextTracksParams)
     public function getRecordingTextTracks($getRecordingTextTracksParams)
     {
         return new GetRecordingTextTracksResponse(
-            $this->processJsonResponse($this->getRecordingTextTracksUrl($getRecordingTextTracksParams))
+            $this->processResponse($this->getRecordingTextTracksUrl($getRecordingTextTracksParams))
```

Comment:
I removed the processJsonResponse function, instead I created a processResponse that can be used to get json data and that is used by the processXmlResponse function.

Reply:
@SamuelWei  that's good remove duplicate codes thanks

Reply:
@joisarjignesh See my latest commit. I re-added the processJsonResponse, but right now it's more or less just an alias.